app/models/calculation_result.rb

```diff
@@ -285,17 +285,22 @@ def main_home_rows(prefix:)
     {
       value: monetise(capital.dig(:main_home, :value)),
       mortgage: monetise(-capital.dig(:main_home, :outstanding_mortgage)),
-      disregard: monetise(-capital.dig(:main_home, :main_home_equity_disregard)),
-    }
+      disregards: monetise(-capital.dig(:main_home, :main_home_equity_disregard)),
+    }.tap do |ret|
+      transaction_allowance = capital.dig(:main_home, :transaction_allowance).to_d
+      ret[:deductions] = monetise(-transaction_allowance) if transaction_allowance.positive?
+    end
```

Comment:
would merge be more idiomatic? [Unsure]

Reply:
Interestingly, Rubocop complains about a RedundantMerge and tells me to do a simple assignment instead. But `tap` isn't doing much here so I've removed it to make things clearer.